doc/howtos/software-update-server.rst

```diff
@@ -0,0 +1,57 @@
+.. _software-update-server:
+
+Software Update: Building an Ostro |trade| OS  Repository
+#########################################################
+
+This technical note provides instructions supporting the creation
+of a repository for feeding updates to devices based on Ostro OS.
+
+Prerequisites
+=============
+
+- additional (fast) disk space: the SW Update tool is provided as
+  additional layer, therefore it doesn't require any additional
+  element.
+  However, the process of generating the bundles/updates can be
```

Comment:
The process of generating the bundles and updates can be very disk intensive and dramatic speed improvements come from using a fast disk storage system, including use of SSD and RAID arrays.